<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TricksDialog</class>
 <widget class="QDialog" name="TricksDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enable Tricks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="vlay_tricks">
     <item>
      <layout class="QHBoxLayout" name="hlay_tricks_body">
       <item>
        <layout class="QVBoxLayout" name="vlay_disabled_tricks">
         <item>
          <widget class="QLabel" name="label_for_disabled_tricks">
           <property name="text">
            <string>Disabled Tricks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="disabled_tricks_free_search">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Search</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="disabled_tricks">
           <property name="toolTip">
            <string>test</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionRectVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vlay_tricks_controls">
         <item>
          <spacer name="vspace_tricks_controls_upper">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="disable_trick">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Disable
&lt;--</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="vspace_tricks_controls_middle">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="enable_trick">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Enable
--&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="vspace_tricks_controls_lower">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vlay_enabled_tricks">
         <item>
          <widget class="QLabel" name="label_for_enabled_tricks">
           <property name="text">
            <string>Enabled Tricks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="enabled_tricks_free_search">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Search</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="enabled_tricks">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="selectionRectVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bbox_tricks">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bbox_tricks</sender>
   <signal>accepted()</signal>
   <receiver>TricksDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bbox_tricks</sender>
   <signal>rejected()</signal>
   <receiver>TricksDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
